src/SDKs/Monitor/Management.Monitor/Generated/Management/Monitor/Models/MetricSettings.cs

```diff
@@ -33,16 +33,11 @@ public partial class MetricSettings
         /// format.</param>
         /// <param name="enabled">a value indicating whether this timegrain is
         /// enabled.</param>
-        /// <param name="category">Name of a Diagnostic Metric category for a
-        /// resource type this setting is applied to. To obtain the list of
-        /// Diagnostic metric categories for a resource, first perform a GET
-        /// diagnostic settings operation.</param>
         /// <param name="retentionPolicy">the retention policy for this
         /// timegrain.</param>
-        public MetricSettings(System.TimeSpan timeGrain, bool enabled, string category = default(string), RetentionPolicy retentionPolicy = default(RetentionPolicy))
+        public MetricSettings(System.TimeSpan timeGrain, bool enabled, RetentionPolicy retentionPolicy = default(RetentionPolicy))
         {
             TimeGrain = timeGrain;
-            Category = category;
```

Comment:
Is this change intentional? I guess it has nothing to do with removal of diagnosticsettings api, that is why just wanted to confirm.